src/Static.js

```diff
@@ -28,7 +28,8 @@ class Static extends React.Component {
       importSettingsTitle: 'Import settings',
       errorDescription: 'Something went wrong when importing your custom setting file. Please try again or proceed with default settings.',
       signInTitle: this.props.signInTitle,
-      signInStatus: this.props.signInStatus
+      signInStatus: this.props.signInStatus,
+      importSettingsTitle: this.props.importSettingsTitle,
```

Comment:
Duplicate key importSettingsTitle

Reply:
I think when @filipeotero created the first version, he meant to have a default value and then value can also be passed from Dynamo. Please check

Reply:
removed the hardcoded string